releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -288,6 +288,7 @@ private void createAllIssues(String... labels) {
                 + "LongAndWeExpectItToGetWrappedDueToBeingTooLng", label);
             addIssue(9, CLOSED, "Mock issue title 9 escape @ and @@@@@", label);
             addIssue(10, CLOSED, "Mock issue title 10 escape < > & <&<>&<<", label);
+            addIssue(13, CLOSED, "Mock pull title 13", label);
```

Comment:
@rnveach Since I added a commit starting with `Pull #13:`, I had to add a corresponding issue to get rid of the warning that it does not exist. However, this does not feel right, since I am creating a pull request and not an issue technically. For now, it works as intended, but I think we can extend `AbstractReleaseNotesTestSupport` with `addPullRequest()` for instance.

Reply:
The reason it works is because to the GH api it is all the same. https://github.com/checkstyle/contribution/blob/2617fbafa32323ef3c3a4b25172388451177c8c1/releasenotes-builder/src/main/java/com/github/checkstyle/github/NotesBuilder.java#L148

Anything like `addPullRequest` will just be a wrap to `addIssue`.

Reply:
Alright, thanks for the clarification!